2018-07-19 11:42:57.4423|INFO|Info|User Iserial:537
2018-07-19 11:42:57.4583|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_TBPo1_TBLITEM\". The conflict occurred in database \"ccnew\", table \"dbo.TBLITEMprice\", column 'ISerial'.\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite)\r\n   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)\r\n   at System.Data.EntityClient.EntityAdapter.Update(IEntityStateManager entityCache)\r\n   at System.Data.Objects.ObjectContext.SaveChanges(SaveOptions options)\r\n   at System.Data.Objects.ObjectContext.SaveChanges()\r\n   at CCWFM.Web.Service.CRUD_ManagerService.PostRetailPoDetail(TblStyle style, TblSalesOrder salesOrder, Int32 retailbrand, Int32 styleIserial, TblSalesOrderSizeRatio size, Int32 poheader, Int32 Dserial, Nullable`1 deliveryDate, TblSalesOrderColor color) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\PostStyleToRetail.cs:line 436\r\n   at CCWFM.Web.Service.CRUD_ManagerService.PostStyleToPo(Int32 iserial, Int32 salesOrder, Boolean postPo, Int32 tblAuthUser, Nullable`1& tempprice, Nullable`1& tempCost) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\PostStyleToRetail.cs:line 235\r\n   at CCWFM.Web.Service.CRUD_ManagerService.UpdateOrInsertTblApproval(TblApproval newRow, Int32 index, Int32 perm, Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\StyleApproval.cs:line 69\r\n   at SyncInvokeUpdateOrInsertTblApproval(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndUpdateOrInsertTblApproval(Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndUpdateOrInsertTblApproval(Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndUpdateOrInsertTblApproval(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.UpdateOrInsertTblApprovalCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.ApprovalViewModel.<.ctor>b__13_8(Object s, UpdateOrInsertTblApprovalCompletedEventArgs x)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnUpdateOrInsertTblApprovalCompleted(Object state)"}
2018-07-19 11:48:19.0401|INFO|Info|User Iserial:537
2018-07-19 11:48:19.0401|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_TBPo1_TBLITEM\". The conflict occurred in database \"ccnew\", table \"dbo.TBLITEMprice\", column 'ISerial'.\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite)\r\n   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)\r\n   at System.Data.EntityClient.EntityAdapter.Update(IEntityStateManager entityCache)\r\n   at System.Data.Objects.ObjectContext.SaveChanges(SaveOptions options)\r\n   at System.Data.Objects.ObjectContext.SaveChanges()\r\n   at CCWFM.Web.Service.CRUD_ManagerService.PostRetailPoDetail(TblStyle style, TblSalesOrder salesOrder, Int32 retailbrand, Int32 styleIserial, TblSalesOrderSizeRatio size, Int32 poheader, Int32 Dserial, Nullable`1 deliveryDate, TblSalesOrderColor color) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\PostStyleToRetail.cs:line 436\r\n   at CCWFM.Web.Service.CRUD_ManagerService.PostStyleToPo(Int32 iserial, Int32 salesOrder, Boolean postPo, Int32 tblAuthUser, Nullable`1& tempprice, Nullable`1& tempCost) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\PostStyleToRetail.cs:line 235\r\n   at CCWFM.Web.Service.CRUD_ManagerService.UpdateOrInsertTblApproval(TblApproval newRow, Int32 index, Int32 perm, Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\StyleApproval.cs:line 69\r\n   at SyncInvokeUpdateOrInsertTblApproval(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndUpdateOrInsertTblApproval(Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndUpdateOrInsertTblApproval(Int32& outindex, Nullable`1& tempprice, Nullable`1& tempCost, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndUpdateOrInsertTblApproval(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.UpdateOrInsertTblApprovalCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.ApprovalViewModel.<.ctor>b__13_8(Object s, UpdateOrInsertTblApprovalCompletedEventArgs x)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnUpdateOrInsertTblApprovalCompleted(Object state)"}
2018-07-19 14:22:27.5523|INFO|Info|User Iserial:5
2018-07-19 14:22:27.5683|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.SharedOperation.GetSqlConnectionString(String dbName) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\SharedOperation.cs:line 214\r\n   at CCWFM.Web.Service.AuthOp.AuthService.GetAuthJournalSetting(Int32 userIserial, String company) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\AuthOp\\AuthUserJournalSetting.cs:line 22\r\n   at SyncInvokeGetAuthJournalSetting(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.AuthServiceClientChannel.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.CCWFM.AuthService.AuthService.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.OnEndGetAuthJournalSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.AuthService.GetAuthJournalSettingCompletedEventArgs.get_Result()"}
2018-07-19 14:35:05.5043|INFO|Info|User Iserial:5
2018-07-19 14:35:05.5193|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.SharedOperation.GetSqlConnectionString(String dbName) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\SharedOperation.cs:line 214\r\n   at CCWFM.Web.Service.AuthOp.AuthService.GetAuthJournalSetting(Int32 userIserial, String company) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\AuthOp\\AuthUserJournalSetting.cs:line 22\r\n   at SyncInvokeGetAuthJournalSetting(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.AuthServiceClientChannel.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.CCWFM.AuthService.AuthService.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.OnEndGetAuthJournalSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.AuthService.GetAuthJournalSettingCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.UserJobsViewModel.<.ctor>b__1_10(Object s, GetAuthJournalSettingCompletedEventArgs e)\r\n   at CCWFM.AuthService.AuthServiceClient.OnGetAuthJournalSettingCompleted(Object state)"}
2018-07-19 14:35:18.8803|INFO|Info|User Iserial:5
2018-07-19 14:35:18.8803|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.SharedOperation.GetSqlConnectionString(String dbName) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\SharedOperation.cs:line 214\r\n   at CCWFM.Web.Service.AuthOp.AuthService.GetAuthJournalSetting(Int32 userIserial, String company) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\AuthOp\\AuthUserJournalSetting.cs:line 22\r\n   at SyncInvokeGetAuthJournalSetting(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.AuthServiceClientChannel.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.CCWFM.AuthService.AuthService.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.OnEndGetAuthJournalSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.AuthService.GetAuthJournalSettingCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.UserJobsViewModel.<.ctor>b__1_10(Object s, GetAuthJournalSettingCompletedEventArgs e)\r\n   at CCWFM.AuthService.AuthServiceClient.OnGetAuthJournalSettingCompleted(Object state)"}
2018-07-19 14:35:30.0723|INFO|Info|User Iserial:5
2018-07-19 14:35:30.0723|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.SharedOperation.GetSqlConnectionString(String dbName) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\SharedOperation.cs:line 214\r\n   at CCWFM.Web.Service.AuthOp.AuthService.GetAuthJournalSetting(Int32 userIserial, String company) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\AuthOp\\AuthUserJournalSetting.cs:line 22\r\n   at SyncInvokeGetAuthJournalSetting(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.AuthServiceClientChannel.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.CCWFM.AuthService.AuthService.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.OnEndGetAuthJournalSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.AuthService.GetAuthJournalSettingCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.UserJobsViewModel.<.ctor>b__1_10(Object s, GetAuthJournalSettingCompletedEventArgs e)\r\n   at CCWFM.AuthService.AuthServiceClient.OnGetAuthJournalSettingCompleted(Object state)"}
2018-07-19 14:51:11.4933|INFO|Info|User Iserial:5
2018-07-19 14:51:11.6253|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.SharedOperation.GetSqlConnectionString(String dbName) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\Operations\\SharedOperation.cs:line 214\r\n   at CCWFM.Web.Service.AuthOp.AuthService.GetAuthJournalSetting(Int32 userIserial, String company) in C:\\Users\\Osama.Gamal\\Desktop\\CCWFM\\CCWFM.Web\\Service\\AuthOp\\AuthUserJournalSetting.cs:line 22\r\n   at SyncInvokeGetAuthJournalSetting(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.AuthServiceClientChannel.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.CCWFM.AuthService.AuthService.EndGetAuthJournalSetting(IAsyncResult result)\r\n   at CCWFM.AuthService.AuthServiceClient.OnEndGetAuthJournalSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.AuthService.GetAuthJournalSettingCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.UserJobsViewModel.<.ctor>b__1_10(Object s, GetAuthJournalSettingCompletedEventArgs e)\r\n   at CCWFM.AuthService.AuthServiceClient.OnGetAuthJournalSettingCompleted(Object state)"}
2018-07-19 16:26:43.1653|INFO|Info|User Iserial:5
2018-07-19 16:26:43.1853|ERROR|Info|{"Message":"Value cannot be null.\r\nParameter name: source","Data":{},"InnerException":null,"StackTrace":"   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)\r\n   at Omu.ValueInjecter.Silverlight.DescriptorTools.GetByName(PropertyInfo[] collection, String name)\r\n   at Omu.ValueInjecter.Silverlight.SameNameType.Inject(Object source, Object target)\r\n   at Omu.ValueInjecter.Silverlight.ValueInjection.Map(Object source, Object target)\r\n   at Omu.ValueInjecter.Silverlight.StaticValueInjecter.InjectFrom[T](Object target, Object[] source)\r\n   at Omu.ValueInjecter.Silverlight.StaticValueInjecter.InjectFrom(Object target, Object[] source)\r\n   at CCWFM.ViewModel.Gl.JournalSettingViewModel.<.ctor>b__0_12(Object s, GetTblJournalSettingCostCenterCompletedEventArgs sv)\r\n   at CCWFM.GlService.GlServiceClient.OnGetTblJournalSettingCostCenterCompleted(Object state)"}
